Ontology/Space/Building/Building.json

```diff
@@ -2,6 +2,14 @@
   "@id": "dtmi:digitaltwins:rec_3_3:core:Building;1",
   "@type": "Interface",
   "contents": [
+    {
+      "@type": "Component",
+      "displayName": {
+        "en": "has address"
+      },
+      "name": "hasAddress",
```

Comment:
Can we just use the name **address** instead of **hasAddress**?

We've been using the "has"/"is" for relationships but this is more of a property. It would be a lot nicer when we fetch this twin for a model-driven UI to show "address" instead of "has address".

Reply:
Agreed. Will update OWL models and regenerate. By Friday at the latest.

Reply:
Sorted in updated PR just now.